tests/install-errors/task.yaml

```diff
@@ -1,4 +1,22 @@
 summary: Checks for cli errors installing snaps
+environment:
+  SIDELOAD_SNAP_NAME: basic-binaries
+  STORE_SNAP_NAME: hello-world
+  SNAP_FILE: "./$[SIDELOAD_SNAP_NAME]_1.0_all.snap"
+
+prepare: |
```

Comment:
@zyga yep :) exactly that conversation is the inspiration for this, spread made it possible. i'm very glad you like it, thanks a lot!